tests/documents_tests.ts

```diff
@@ -200,6 +215,28 @@ describe('Documents tests', () => {
         }
       })
 
+      test(`${permission} key: Update partial document in batch`, async () => {
```

Comment:
`Partially update a document in batch` or `Partial update of a document in batch`